sdk/tests/tutorials/ibor/test_orders.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+import uuid
+import lusid
+import lusid.models as models
+from lusid import OrderRequest
+from lusid import OrderSetRequest
+from lusid import PerpetualProperty
+from lusid import PropertyValue
+from lusid import ResourceId
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+
+class Orders(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = TestDataUtilities.api_client()
+        cls.orders_api = lusid.OrdersApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+ #   @lusid_feature("F4")
+    def test_upsert_simple_order(self):
+        """Makes a request for a single order."""
+
+        orders_scope = "Orders-SimpleUpsert-TestScope"
+        # Create unique order id
+        order = str(uuid.uuid4())
+        # Create ResourceId for order
+
+        # Construct order arguments
+        instrument_identifiers = {TestDataUtilities.lusid_luid_identifier: self.instrument_ids[0]}
+        order_id = models.ResourceId(orders_scope, order)
```

Comment:
Rename `order_id` to `order_resource_id` and then you can rename `order` on line 30 to `order_id`. Calling it order is a bit confusing as you'd assume it's a full order object and not just the id